indico/modules/events/timetable/controllers/manage.py

```diff
@@ -165,6 +172,43 @@ def _process_POST(self, data: SessionBlockSchema):
         return TimetableEntrySchema().jsonify(block_entry)
 
 
+class RHAPIEditEntryTime(RHManageTimetableEntryBase):
```

Comment:
Since this seems to be pretty much exactly a copy of RHLegacyTimetableEditEntryDateTime I would really prefer if we kept using the existing RHs for now. It's easy to clean up things later, but adding duplicate code now has only disadvantages IMHO:

- having to check later if it is really duplicate and still has the same behavior
- more noise in PRs

If you need different return values fine, but even then I would consider **for now** to just add a `new_timetable=1` arg that changes what the existing RH returns. It will be trivial to clean up later.

Reply:
Actually, I see that you changed how the args are retrieved.

I would use `_process_args` for this:

- In the original one, get it from `request.form`
- In the new one, you (only) override `_process_args` and decorate it with use_kwargs

(in both places you set `self.start_dt` etc, and then use that in `_process`)